storage-account.tf

```diff
@@ -54,3 +54,11 @@ resource "azurerm_storage_container" "sscs_container" {
   resource_group_name  = "${azurerm_storage_account.storage_account.resource_group_name}"
   storage_account_name = "${azurerm_storage_account.storage_account.name}"
 }
+
+output "storage_account_name" {
+  value = "${azurerm_storage_account.storage_account.name}"
```

Comment:
could you please add the sensitive flag as we don't wan't to see them in Jenkins log ?

Reply:
the flag doesn't do anything